src/main/java/io/debezium/connector/vitess/VitessConnectorConfig.java

```diff
@@ -216,6 +232,28 @@ public Duration getKeepaliveInterval() {
         return getConfig().getDuration(KEEPALIVE_INTERVAL_MS, ChronoUnit.MILLIS);
     }
 
+    public Map<String, String> getGrpcHeaders() {
+        String grpcHeaders = getConfig().getString(GRPC_HEADERS);
+
+        if (grpcHeaders == null) {
+            return Collections.emptyMap();
+        }
+
+        Map<String, String> grpcHeadersMap = new HashMap<>();
+
+        for (String header : grpcHeaders.split(",")) {
+            String[] keyAndValue = header.split(":");
+            if (keyAndValue.length == 2) {
+                grpcHeadersMap.put(keyAndValue[0], keyAndValue[1]);
+            }
+            else {
+                LOGGER.warn("The following gRPC header is invalid: {}", header);
```

Comment:
I intend towards skipping since these are optional headers. I saw a similar pattern for Postgres: https://github.com/debezium/debezium/blob/main/debezium-connector-postgres/src/main/java/io/debezium/connector/postgresql/connection/PostgresReplicationConnection.java#L721

I will update the Debezium doc for this new config.